lib/code_corps/github/adapters/issue.ex

```diff
@@ -54,6 +54,12 @@ defmodule CodeCorps.GitHub.Adapters.Issue do
     payload |> MapTransformer.transform(@task_mapping)
   end
 
+  def to_issue_attrs(%GithubIssue{} = github_issue) do
+    github_issue
+    |> Map.from_struct
+    |> MapTransformer.transform_inverse(@issue_mapping)
+  end
```

Comment:
The naming makes sense, but considering we were switching already, I'm thinking maybe we want to write a behavior module and document what an adapter needs to look like and what the naming convention is.

I'm just imagining where, in a couple of weeks, I write a new one and end up using a new scheme because I wasn't 100% what the old one is.